### DISCUSSION TITLE
How to hide the labels of taskbar items?

### DISCUSSION THREAD

**Discussion by aseem-thakkar** [valinet/ExplorerPatcher#1533] [Q&A]
Hi, I want to hide the labels on taskbar on Win11 using ExplorerPatch, (when using windows10 taskbar). I want to decombine icons and hide labels with 7tt taskbar

Nevermind figured it out.

**Comment by jaburmester** [comment]
How was this done?

**Comment by pyrates999** [comment]
One way is to use a windhawk mod: https://windhawk.net/mods/taskbar-grouping
